packages/injected/src/icons/rabby.ts

```diff
@@ -0,0 +1,24 @@
+export default `<svg width="320" height="320" viewBox="0 0 320 320" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
@vvvvvv1vvvvvv We like to have the icons with 100% width and height, thanks!
```suggestion
export default `<svg width="100%" height="100%" viewBox="0 0 320 320" fill="none" xmlns="http://www.w3.org/2000/svg">
```